server/graphql/requisition/src/lib.rs

```diff
@@ -62,12 +62,13 @@ impl RequisitionQueries {
         get_supplier_program_requisition_settings(ctx, &store_id)
     }
 
-    pub async fn customer_program_requisition_settings(
+    pub async fn program_requisition_settings_by_customer(
         &self,
         ctx: &Context<'_>,
         store_id: String,
-    ) -> Result<Vec<CustomerProgramRequisitionSettingNode>> {
-        get_customer_program_requisition_settings(ctx, &store_id)
+        customer_store_id: String,
+    ) -> Result<CustomerProgramRequisitionSettingNode> {
+        get_program_requisition_settings_by_customer(ctx, &store_id, &customer_store_id)
```

Comment:
Now we pass through customer_name_id and available programs on this rather than store_id